presigned-pause.just

```diff
@@ -156,3 +156,66 @@ simulate-all:
       --json-file ${taskPath}/tx/draft-${nonce}.json \
       simulate
   done
+
+upload-PSPs:
+ #!/bin/bash
+ # Navigate to the directory containing the JSON files
+ # Uncomment and modify the next line if you need to change directories
+ echo "=================== PSPs Uploader ============================="
+ echo "This is about to Upload the PSPs (ready-X.sh.b64 and ready-X.json) to 1Password"
+ echo "Please make sure you have the files in the tx/ folder, if the PSPs is already uploaded this will duplicates it."
```

Comment:
To clarify this behavior—we do not overwrite existing PSPs, and instead would have two files named `ready-X.sh.b6`?
```suggestion
 echo "Please make sure you have the files in the tx/ folder, if the PSPs is already uploaded this will duplicate it."
```

Reply:
@mds1 yes this is correct the files are duplicates in case we are doing a bad actions this one is going to be duplicated :) Thanks for typos correction!